insights/specs/default.py

```diff
@@ -369,6 +370,7 @@ class DefaultSpecs(Specs):
     lspci_vmmkn = simple_command("/sbin/lspci -vmmkn")
     lsscsi = simple_command("/usr/bin/lsscsi")
     lsvmbus = simple_command("/usr/sbin/lsvmbus -vv")
+    luksmeta = foreach_execute(cryptsetup.luks1_block_devices, "luksmeta show -d %s")
```

Comment:
BTW, please use the full command path for the `luksmeta` command (/usr/bin/luksmeta) in the spec, since the insights-client may not get the correct PATH environment.


And, Would you please modify the following two specs to use the full path (/usr/sbin/cryptsetup)? https://github.com/RedHatInsights/insights-core/blob/534888a20d32c256dd1443fe8b281c9927e4054d/insights/specs/datasources/luks_devices.py#L43-L44

And sorry for missing this in the review of that PR.